example/data_source_virtual_environment_metrics_server.tf

```
@@ -0,0 +1,3 @@
+data "proxmox_virtual_environment_metrics_server" "example" {
+  name = "example_influxdb_server"
```

```suggestion
  name = proxmox_virtual_environment_metrics_server.influxdb_server.name
```
this will establish a dependency between the datasource and the resource, so TF will ensure that resource is created first before reading the datasource.

---

got it, done

although i suspect that adding new stuff in "example" folder is deprecated? or isn't it? i added acctests and having  even more stuff in example folder looks a bit redundant, what do you think?

---

yeah, for net new functionality we can skip adding to `/example` if there is/are relevant acceptance test(s). 

i'm running both `make example` and `make testacc` before the release